builder/llpcBuilderImplArith.cpp

```diff
@@ -78,6 +78,110 @@ Value* BuilderImplArith::CreateCubeFaceIndex(
     return CreateIntrinsic(Intrinsic::amdgcn_cubeid, {}, { pCoordX, pCoordY, pCoordZ }, nullptr, instName);
 }
 
+// =====================================================================================================================
+// Create scalar or vector FP truncate operation with the given rounding mode.
+// Currently only implemented for float/double -> half conversion.
```

Comment:
how is it handled for double -> float conversion after your change?

Reply:
Good point. I'll add it. :-)